mscore/musescore.cpp

```diff
@@ -1202,6 +1202,26 @@ MuseScore::MuseScore()
 
       populateNoteInputMenu();
 
+      //-------------------------------
+      //    Feedback Tool Bar
+      //-------------------------------
+
+      feedbackTools = new QToolBar("", this);
+      feedbackTools->setObjectName("feedback-tools");
+      // Add the toolbar to the bottom and forbid to move it...
```

Comment:
Why? I really hate it down there, it is eating extra space that now is missing from the scoreview

Reply:
Forbid to move or undock might be the right thing to do though